.github/workflows/buildAndTestCMake.yml

```diff
@@ -84,16 +71,11 @@ jobs:
     - name: Build and Test StableHLO (with AddressSanitizer)
       shell: bash
       run: |
-        ./build_tools/github_actions/ci_build_cmake.sh "$LLVM_BUILD_DIR" "$STABLEHLO_BUILD_DIR"
-      env:
-          CMAKE_BUILD_TYPE: Release
-          STABLEHLO_ENABLE_BINDINGS_PYTHON: OFF
-          STABLEHLO_ENABLE_SANITIZER: address
+        cmake --preset debug
+        cmake --build ./build --target check-stablehlo-ci
```

Comment:
CMake has a concept of a "workflow" that can do this but there is some weird history to CMakePresets [1]

[1] https://gitlab.kitware.com/cmake/cmake/-/issues/22538